<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsynchronousGrab</class>
 <widget class="QDialog" name="AsynchronousGrab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_ButtonStartStop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Image Acquisition</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ColorProcessingCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ColorProcessing</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelStream">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>471</width>
     <height>339</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="m_ListBoxCameras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="m_ListLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>891</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentParameters">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
